plugins/generator-1.19.4/forge-1.19.4/utils/aiconditions.java.ftl

```diff
@@ -0,0 +1,32 @@
+<#include "procedures.java.ftl">
+<#-- @formatter:off -->
+<#macro conditionCode conditionfield="" includeBractets=true>
+    <#if conditionfield?has_content>
+        <#assign conditions = generator.procedureNamesToObjects(conditionfield)>
+        <#if hasProcedure(conditions[0]) || hasProcedure(conditions[1])>
+			<#if includeBractets>{</#if>
+                <#if hasProcedure(conditions[0])>
+                @Override public boolean canUse() {
+                	double x = ${name}Entity.this.getX();

```

Comment:
The indentations in this file are very strange. Please recheck them and also other files.

Reply:
I did not edit this, I just copied it

Reply:
When copying files using IDEA, IDEA sometimes messes up indentations.

also whitespace and tab use should be as unifed as possible (we use tabs)